website/blog/2023-09-26-how-to-contribute-to-open-source-2022.md

```diff
@@ -25,24 +25,25 @@ This is how the members of the open source community encourage new tech enthusia
 
 [Hacktoberfest](https://hacktoberfest.com/) is an annual online event celebrating the open source movement.
 This year, registration starts on September 26.
-To participate, you need to send at least four open-sourced pull requests on GitHub or Gitlab between October 1st and October 31st, and those pull requests need to be accepted by the repository owners.
+To participate, you need to send at least four open-sourced pull requests on GitHub or Gitlab between October 1st and October 31st, for projects that indicate they are looking for Hacktoberfest contributions (`hacktoberfest` labels), and those pull requests need to be accepted by the repository owners.
```

Comment:
The dates are different. `hacktoberfest` is a repository topic, not a label. Repositories have [topics](https://github.com/topics/hacktoberfest), issues have [labels](https://github.com/FerretDB/FerretDB/labels/hacktoberfest).